release notes/v0.47.0.md

```diff
@@ -37,6 +37,32 @@ Thanks, @sapphire-janrain, for contribution!
 
 In some workflows the reflection call should also include some metadata. This PR adds [a new connection parameter `reflectMetadata`](https://k6.io/docs/javascript-api/k6-net-grpc/client/client-connect/#connectparams) that allows to specify the metadata to be sent with the reflection call.
 
+### Improved the browser module's cookie API
+
+The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) that can be used while creating and retrieving cookies.
```

Comment:
This suggestion is linked to https://github.com/grafana/k6/pull/3352/files#r1337145487

```suggestion
The browser module now provides a more complete and robust API for handling cookies. We stabilized the cookie API by defining a new [`Cookie` class](https://k6.io/docs/javascript-api/k6-experimental/browser/browsercontext/cookie) ([browser#1008](https://github.com/grafana/xk6-browser/pull/1008), [browser#1030](https://github.com/grafana/xk6-browser/pull/1030)) that can be used while creating and retrieving cookies.
```

Reply:
Ah, I see now. You want to make the related PRs clearer 👍